bin/test_end_to_end.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash -l
 
 #PBS -P iq82
-#PBS -l storage=gdata/xp65+gdata/ik11+gdata/cj50+gdata/p73+gdata/dk92+gdata/al33+gdata/rr3+gdata/fs38+gdata/oi10
+#PBS -l storage=gdata/xp65+gdata/ik11+gdata/cj50+gdata/p73+gdata/dk92+gdata/al33+gdata/rr3+gdata/fs38+gdata/oi10+gdata/hq89+gdata/py18+gdata/ig45+gdata/zz63+gdata/rt52
```

Comment:
I'm not sure why this is flagging as a change, given these updates already exist in `main`... is this branch up-to-date with `main`?

Reply:
This is on the e2e, not the build_all script - I've just updated the flags on `test_end_to_end.sh` to match `build_all.sh`

Reply:
You're right, so it is. Ignore me.